### ISSUE TITLE
Display exercise identifiers on the web interface

### ISSUE THREAD

**Issue by yurug** [ocaml-sf/learn-ocaml#78]
To use the command line client, an exercise identifier is needed. Yet, as far as I can tell, the web interface does not display this identifier.

Would it make sense to also update the command line client to get the student current homework assignments? Something like:
```
learn-ocaml-client assignments
```
would display a two-columns list like:
```
exercise-identifier-0      http://learn-ocaml-server/url-to-exercise-0-statement
exercise-identifier-1      http://learn-ocaml-server/url-to-exercise-1-statement
exercise-identifier-2      http://learn-ocaml-server/url-to-exercise-2-statement
```

**Comment by AltGr** [comment]
Good suggestion, yes.
The only mention of the exercise id, otherwise, is in (the middle of) the address bar... hardly convenient.

**Comment by yurug** [comment]
This issue starts to matter since Paris Diderot will start using the learn-ocaml-client tool next week.

**Comment by AltGr** [comment]
Fixed by #89
